helix-core/src/main/java/org/apache/helix/task/TaskDriver.java

```diff
@@ -1065,16 +1066,16 @@ private void setSingleWorkflowTargetState(String workflow, final TargetState sta
 
     WorkflowConfig workflowConfig = TaskUtil.getWorkflowConfig(_accessor, workflow);
     if (workflowConfig == null) {
-      LOG.warn("WorkflowConfig for " + workflow + " not found!");
+      LOG.warn("WorkflowConfig for {} not found!", workflow);
       return;
     }
 
     WorkflowContext workflowContext = TaskUtil.getWorkflowContext(_propertyStore, workflow);
     if (state != TargetState.DELETE && workflowContext != null
         && workflowContext.getFinishTime() != WorkflowContext.UNFINISHED) {
       // Should not update target state for completed workflow
-      LOG.info("Workflow " + workflow + " is already completed, skip to update its target state "
-          + state);
+      LOG.info("Workflow {} is already completed, skip to update its target state {}",
+          workflow + state);
```

Comment:
This could cause runtime exception. Not workflow + state.

Reply:
TFTR. Updated. It was a missed typo I have. Thanks for catching this.